src/Apps/Order2/Routes/Checkout/Components/OfferStep/Order2OfferStep.tsx

```diff
@@ -1,43 +1,71 @@
 import { ContextModule } from "@artsy/cohesion"
 import { Box, Button, Flex, Spacer, Text, TextArea } from "@artsy/palette"
 import { validateAndExtractOrderResponse } from "Apps/Order/Components/ExpressCheckout/Util/mutationHandling"
-import { appendCurrencySymbol } from "Apps/Order/Utils/currencyUtils"
 import {
   CheckoutStepName,
   CheckoutStepState,
 } from "Apps/Order2/Routes/Checkout/CheckoutContext/types"
-import { Order2ExactPriceOfferForm } from "Apps/Order2/Routes/Checkout/Components/OfferStep/Forms/Order2ExactPriceOfferForm"
-import { Order2HiddenPriceOfferForm } from "Apps/Order2/Routes/Checkout/Components/OfferStep/Forms/Order2HiddenPriceOfferForm"
-import { Order2PriceRangeOfferForm } from "Apps/Order2/Routes/Checkout/Components/OfferStep/Forms/Order2PriceRangeOfferForm"
-import { Order2OfferCompletedView } from "Apps/Order2/Routes/Checkout/Components/OfferStep/Order2OfferCompletedView"
+import {
+  CheckoutErrorBanner,
+  type CheckoutErrorBannerProps,
+  MailtoOrderSupport,
+} from "Apps/Order2/Routes/Checkout/Components/CheckoutErrorBanner"
+import { OfferInput } from "Apps/Order2/Routes/Checkout/Components/OfferStep/Components/OfferInput"
+import { Order2OfferOptions } from "Apps/Order2/Routes/Checkout/Components/OfferStep/Components/Order2OfferOptions"
+import {
+  Order2OfferCompletedView,
+  type Order2OfferCompletedViewProps,
+} from "Apps/Order2/Routes/Checkout/Components/OfferStep/Order2OfferCompletedView"
 import type { OfferNoteValue } from "Apps/Order2/Routes/Checkout/Components/OfferStep/types"
 import { useCompleteOfferData } from "Apps/Order2/Routes/Checkout/Components/OfferStep/useCompleteOfferData"
 import { useCheckoutContext } from "Apps/Order2/Routes/Checkout/Hooks/useCheckoutContext"
 import { useOrder2AddInitialOfferMutation } from "Apps/Order2/Routes/Checkout/Mutations/useOrder2AddInitialOfferMutation"
 import { useOrder2UnsetOrderFulfillmentOptionMutation } from "Apps/Order2/Routes/Checkout/Mutations/useOrder2UnsetOrderFulfillmentOptionMutation"
 import { mostRecentCreatedAt } from "Apps/Order2/Routes/Checkout/Utils/mostRecentCreatedAt"
-import { useJump } from "Utils/Hooks/useJump"
 import createLogger from "Utils/logger"
-import type { Order2OfferStep_order$key } from "__generated__/Order2OfferStep_order.graphql"
+import type {
+  Order2OfferStep_order$data,
+  Order2OfferStep_order$key,
+} from "__generated__/Order2OfferStep_order.graphql"
 import type { useOrder2AddInitialOfferMutation$data } from "__generated__/useOrder2AddInitialOfferMutation.graphql"
-import { useMemo, useState } from "react"
+import { Formik, type FormikConfig, useFormikContext } from "formik"
+import { useState } from "react"
 import { graphql, useFragment } from "react-relay"
+import * as yup from "yup"
 
 const logger = createLogger(
   "Order2/Routes/Checkout/Components/OfferStep/Order2OfferStep.tsx",
 )
 
-export const DEFUALT_OFFER_NOTE_PREFIX = "I sent an offer for"
```

Comment:
I’m removing the default note for right now. This is causing issues bc it needs to be hidden from the UI, but there is no way to distinguish a default note from a custom note in our backend and I don't want to rely on comparing strings. I’m going to have a conversation about it with product bc we may want to go in a different direction here entirely.